test/colony-network-mining.js

```diff
@@ -288,8 +290,8 @@ contract("ColonyNetworkMining", accounts => {
     });
 
     it("should only allow the first submission after the window closes to be accepted", async () => {
-      await giveUserCLNYTokensAndStake(colonyNetwork, MAIN_ACCOUNT, DEFAULT_STAKE);
       await giveUserCLNYTokensAndStake(colonyNetwork, OTHER_ACCOUNT, DEFAULT_STAKE);
+      await advanceMiningCycleNoContest({ colonyNetwork, test: this });
```

Comment:
If this is added here shouldn't we remove the next 3 lines?

Reply:
Hmm, looking at this test more closely we may be able to do away with this setup entirely, as the modifier which checks if the submission window is open comes before the one which checks the stake and timestamp.